docker-compose.yml

```diff
@@ -37,12 +37,11 @@ services:
       - MONGO_URL=mongodb://mongo:27017/rocketchat
       - MONGO_OPLOG_URL=mongodb://mongo:27017/local
       - MAIL_URL=smtp://smtp.email
-      - OVERWRITE_SETTING_Show_Setup_Wizard=completed
       - ADMIN_USERNAME=guby
       - ADMIN_PASSWORD=guby123
       - ADMIN_EMAIL=guby@guby.live
-#       - HTTP_PROXY=http://proxy.domain.com
-#       - HTTPS_PROXY=http://proxy.domain.com
+      - OVERWRITE_SETTING_Show_Setup_Wizard=completed
+#    env_file: .env
```

Comment:
```suggestion
```

Let's remove off this line?

Reply:
But we saw that there is an issue if we do that no?

Reply:
Nope. No issue was there. docker-compose automatically takes the `.env` file. But we were trying the properties inside the `guby_initialize.sh` file as well. So if we add a `source .env` over in the init file, it will all be good and we can get rid of these lines 👍

Reply:
On second thought, let's leave this in here and fix it in another PR altogether.

Reply:
okay great, yes lets do that!